app/AppContainer.js

```diff
@@ -122,19 +122,19 @@ export class AppContainer extends Component {
     // Implementation
     //------------------------
     unauthorizedMessage() {
-        const getUser = XH.getUser;
+        const user = XH.getUser();
+
         return div(
             XH.accessDeniedMessage,
-            br(),
-            `You are user: ${getUser().username}`,
-            br(),
-            `Your current role(s): ${getUser().roles.join(', ') || '--'}`,
-            div({
-                hidden: !XH.appModel.enableLogout,
-                items: [
-                    br(),
-                    logoutButton({text: 'Logout'})
-                ]
+            vspacer(10),
+            `
+                You are logged in as ${user.username} 
+                and have the roles [${user.roles.join(', ') || '--'}].
+            `,
+            vspacer(20),
+            logoutButton({
+                text: 'Logout',
+                omit: !XH.appModel.enableLogout
             })
```

Comment:
@cnrudd  and @lbwexler - I didn't see a way that we would need to be refreshing the user dynamically here after returning this message element, hence the tweak to simplify and put the user object itself in a const.  Let me know if I am missing something here!

I also thought it worth using our `omit` support and `vspacer` elem.